Sources/ARKit-CoreLocation/SceneLocationView.swift

```diff
@@ -138,8 +138,7 @@ public extension SceneLocationView {
 
     func run() {
         // Create a session configuration
-        let configuration = ARWorldTrackingConfiguration()
-        configuration.planeDetection = .horizontal
+        let configuration = AROrientationTrackingConfiguration()
```

Comment:
`ARWorldTrackingConfiguration`: A configuration that monitors the iOS device’s position and orientation, while enabling you to augment the environment that’s in front of the user.
`AROrientationTrackingConfiguration` - A configuration you use when you just want to track the device’s orientation using the device’s back camera.

Great catch, I am going to need to run some tests on this, but at a glance, this seems perfectly reasonable.

Reply:
In our usage case, we get input from CLLocation and CLHeading for the 3DOF.  While 6DOF would be better, there's no plane in the real world to sync to, and in fact if the OS did locate and sync to a plane, that'd would be destructive.